Gopkg.lock

```diff
@@ -377,6 +377,20 @@
   revision = "4b7aa43c6742a2c18fdef89dd197aaae7dac7ccd"
   version = "1.0.1"
 
+[[projects]]
```

Comment:
Hmm, running `dep ensure` pulls in these dependencies once they're referenced, and it updates `Gopkg.lock` but not `Gopkg.toml`. I'm not sure if that's important.